### PR TITLE
AddCity

### PR DESCRIPTION
**Pull request by at-tringuyen3**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-20

## Reference Isuees
#20

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/CityController.php [hunk 1]
```diff
@@ -41,19 +41,29 @@ public function index()
...
-     * @param \Illuminate\Http\Request $request request
+     * @param \Illuminate\Http\CityRequest $request request
      *
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(CityRequest $request)
     {
-        echo $request;
+        // $data = $request->only('city', 'country');
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/CityController.php]: xoá những dòng comment lại đi

File: app/Http/Requests/Admins/CityRequest.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        // $city = $this->route()->parameter('room_type');
+        switch ($this->method()) {
+            case 'POST':
+                return [
+                    'city' => 'required|unique:cities,city',
+                    'country' => 'required',
+                ];
+                break;
+            // case 'PUT':
```
Comment (at-tuannguyen3) [inline app/Http/Requests/Admins/CityRequest.php]: xoá dòng comment lại bỏ trường hợp default vào

File: app/Http/Requests/Admins/CityRequest.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        // $city = $this->route()->parameter('room_type');
```
Comment (at-tuannguyen3) [inline app/Http/Requests/Admins/CityRequest.php]: xoá dòng comment đi

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment! tks you

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**